frontend_lib/src/component/DropdownMenu/DropdownMenu.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import DropdownMenuItem from './DropdownMenuItem.jsx'
+
+const DropdownMenu = props => {
+  return (
+    <div className='dropdown'>
+      <button
+        aria-expanded='false'
+        aria-haspopup='true'
+        className={classnames(
+          'btn dropdown-toggle',
+          props.isButton ? 'nohover primaryColorBorder' : 'transparentButton',
+          props.buttonCustomClass
+        )}
+        data-cy={props.buttonDataCy}
+        data-toggle='dropdown'
+        disabled={props.buttonDisabled}
+        id='dropdownMenuButton'
+        onClick={e => e.stopPropagation()}
+        title={props.buttonTooltip ? props.buttonTooltip : props.buttonLabel}
+        type='button'
+      >
+        {props.buttonOpts}
+        {props.buttonIcon && <i className={`fa fa-fw ${props.buttonIcon}`} />}
+        {props.buttonImage && <img className='dropdownMenu__image' src={props.buttonImage} />}
+        {props.buttonLabel && <span>{props.buttonLabel}</span>}
+      </button>
+
+      <div
+        aria-labelledby='dropdownMenuButton'
+        className={classnames('dropdownMenu dropdown-menu', props.menuCustomClass)}
+      >
+        {props.children.length > 1
+          ? props.children.map(child => <DropdownMenuItem customClass={props.itemCustomClass}> {child} </DropdownMenuItem>)
+          : <DropdownMenuItem customClass={props.itemCustomClass}> {props.children} </DropdownMenuItem>
```

Comment:
You probably need a `key` attribute here. Linting is disabled in frontend_lib IIRC, we can see how you can run them if you need.

Reply:
Updated.